tabpfn_client/client.py

```diff
@@ -170,11 +172,11 @@ def _validate_response(response, method_name, only_version_check=False):
         try:
             load = response.json()
         except json.JSONDecodeError as e:
-            logging.error(f"Failed to parse JSON from response in {method_name}: {e}")
+            logging.info(f"Failed to parse JSON from response in {method_name}: {e}")
```

Comment:
Why changed it to `info` though?

Reply:
As otherwise our outputs look ugly with these messages in them, even though it is not really an error, since it is intended behavior that our ValueErrors are caught that way.